### ISSUE TITLE
[feat] Generic webhook: support GET

### ISSUE THREAD

**Issue by huyz** [containrrr/shoutrrr#448]
Healthchecks.io and Uptime Kuma accept pushes/pings as GET requests.

But Shoutrrr only supports POST requests. Could GET be added? Even if no query parameters are supported.

**Comment by JordyEGNL** [comment]
A little late, but uptime kuma supports POST requests now, and shoutrrr can support GET

`generic://status.domain.tld/api/push/randomstring?template=json`

`?RequestMethod=GET` for GET

**Comment by piksel** [comment]
Yeah, Shoutrrr *does* support GET requests, but it has to be specified using the query parameter `method=GET`. Using `RequestMethod` and `Method` as the query paramter **should** work, but currently doesn't.
